build.sbt

```diff
@@ -2020,7 +2018,10 @@ lazy val e2eTests = (project in file("e2e-tests"))
   .settings {
     // TODO: it'd be better to use scalaVersion here, but for some reason it's hard to disable existing task dynamically
     forScalaVersion(defaultScalaV) {
-      case (2, 12) => doTest
+      case (2, 12) => {
+        System.setProperty("addDevArtifacts", "true")
```

Comment:
This is why `addDevArtifacts` was changed to a def. Instead of just setting it on CI workflow definition, I wanted to have the e2eTests/test sbt task behave the same on CI and when launching locally.

Reply:
If we're adding table components to public distribution its not necessary